.github/workflows/main.yml

```diff
@@ -33,4 +33,4 @@ jobs:
         run: pnpm build
 
       - name: Test
-        run: pnpm test -- --ci --coverage --maxWorkers=2
+        run: pnpm test -- -- --ci --coverage --maxWorkers=2
```

Comment:
Using `pnpm run` instead of `pnpm` should allow you to remove one end of command